### ISSUE TITLE
Question On Oauth Redirect

### ISSUE THREAD

**Issue by nafwa03** [niutech/node.php#24]
Hi,
Great implementation. Love it. I wanted to know if you have any insight on how to handle this situation and if it is possible. In my url for redirect I keep getting '&redirect_uri=https://mydomain/node.php?path:5000/auth'

On my oauth provider level it will not redirect just keeps getting stuck.

**Comment by niutech** [comment]
What is your desired destination URL? Shouldn't https://example.com/node.php?path=/auth work?

**Comment by nafwa03** [comment]
Yes there may have been some confusion. Trying to run it on a different port. Callback capturing but won't redirect. I may have my path piece wrong didn't see route as = first thought resolved path/auth
